src/ocrd/lib.bash

```diff
@@ -44,12 +54,17 @@ ocrd__minversion () {
         # MINOR == MINOR
         elif (( ${version[1]} == ${minversion[1]} ));then
             # PATCH > PATCH
-            if (( ${version[2]} >= ${minversion[2]} ));then
+            if (( ${version[2]} > ${minversion[2]} ));then
+                return
+            elif (( ${version[2]} == ${minversion[2]}));then
+              # Match prerelease suffix like a1, b1 only literally
+              if [[ $version_prerelease_suffix == $minversion_prerelease_suffix ]];then
```

Comment:
What if I specify `3.0.0a1` but we then put out another `3.0.0a2` – shouldn't that match?

```suggestion
              # Match prerelease suffix like a1, b1 alphabetically
              if [[ $version_prerelease_suffix = $minversion_prerelease_suffix -o $version_prerelease_suffix > $minversion_prerelease_suffix ]]; then
```

Reply:
> What if I specify 3.0.0a1 but we then put out another 3.0.0a2 – shouldn't that match?

Realistically, I'd say no because it's likely things change drastically between those versions. But I am totaly fine with sorting alphabetically, I never knew you can compare strings like that in bash.